notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -416,26 +385,26 @@
       "source": [
         "## Introduction to Vertex AI Model Monitoring\n",
         "\n",
-        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skew and drift detection of the features in the inbound prediction requests or skew and drift detection of the feature attributions (Explainable AI) in the outbound prediction response -- that is, the distribution of the attributions on how they contributed to the output (predictions).\n",
+        "Vertex AI Model Monitoring is supported for AutoML tabular models and custom tabular models. You can monitor for skewness and drift detection of the features in the inbound prediction requests or the feature attributions (Explainable AI) in the outbound prediction response. In other words, you monitor the distribution of the attributions that quantify feature contributions to the output (predictions).\n",
```

Comment:
Change "In other words," to "That is, "

- That is, you monitor the distribution...